### DISCUSSION TITLE
Simpler thermal design

### DISCUSSION THREAD

**Discussion by furmek** [stevenbennett/DIYson#47] [Ideas]
Have you considered simplifying led/Al extrusion thermal interface.

this design:
![image](https://github.com/stevenbennett/DIYson/assets/13870359/b629b8fd-04ab-436f-95c1-7b759b92bebb)
has two problems:
1. assuming you are pumping 12W of power and assuming dimensions of the bridge to be 3.2x6x20mm you are looking at ~30°C temp difference on just that little section.
2. Making a M3 hole that cuts out 50% of surface area makes most of the Cu bar be redundant*

* this is assuming that Al extrusion has thermal conductivity at about 250W/mK while Cu at about 400 while extrusion cross section area is about 10x grater than of the Cu bar.

It would be good to get a baseline test where led is directly coupled to the extrusion - this will show the lower limit of what is possible with passive cooling.  
Another interesting test would be to measure temperature not only 200 and 100mm from the end of extrusion but also at the very end - just right next to where Cu meets Al.

As for the simpler solution: make a simple L shape bracket that screws to the end of the extrusion and has as much meat between led and extrusion as possible. From the looks of it your provider can do up to .25' thick copper, that would potentially give you (6.35*20) 127mm² cross section compared to ~20 you have now. That may be even stackable.

**Comment by stevenbennett** [comment]
I see what you mean. I'd like to do these tests too. Alongside a baseline test on bare aluminum, a better bracket wouldn't have the big m3-sized interruptions. That makes sense. And the simple L-shape bracket makes sense too; the construction would be dead-simple and the cost per lamp would also decrease, maybe without much cooling sacrifice.

I'm not sure when I will test these because it's costly for me to order small batches of these parts, but I will keep it in mind. Thanks.
